integration/spark/app/src/test/java/io/openlineage/spark/agent/column/ColumnLevelLineageIcebergTest.java

```diff
@@ -378,7 +380,9 @@ void testCTEQuery() {
     assertColumnDependsOn(facet, "c", FILE, T1_EXPECTED_NAME, "a");
     assertColumnDependsOn(facet, "d", FILE, T1_EXPECTED_NAME, "b");
     assertColumnDependsOnInputs(facet, "c", 1);
-    assertColumnDependsOnInputs(facet, "d", 2);
+    assertColumnDependsOnInputs(facet, "d", 1);
+    assertDatasetDependsOnType(
```

Comment:
Is this the only test affected in the class by dataset depenendency feature? If so, why do we want to create whole `ColumnLevelLineageIcebergOnlyFieldDependenciesTest` file?

Reply:
I chose to copy the files for two reasons. First, to finally remove it, we just delete the entire files without thinking. Second, and most importantly, there is only one place in the tests where the new dataset dependency mode flag is set. Switching it would affect every test. Without hard-to-review to review refactor, it would be impossible to control the flag's value. Additionally, at some point, we'd need to reverse the refactor to avoid causing confusion. The `OnlyFieldDependenciesTest` suites are just copies of what was there so far and are intended for removal.